activemodel/lib/active_model/validations/acceptance.rb

```diff
@@ -3,12 +3,12 @@ module ActiveModel
   module Validations
     class AcceptanceValidator < EachValidator # :nodoc:
       def initialize(options)
-        super({ allow_nil: true, accept: "1" }.merge!(options))
+        super({ allow_nil: true, accept: ["1", true] }.merge!(options))
```

Comment:
Somebody might using "true" in their test cases. Then what is the purpose for this patch?